models/system/setting.go

```diff
@@ -259,41 +259,11 @@ var (
 )
 
 func Init(ctx context.Context) error {
-	var disableGravatar bool
-	disableGravatarSetting, err := GetSetting(ctx, KeyPictureDisableGravatar)
-	if IsErrSettingIsNotExist(err) {
-		disableGravatar = setting_module.GetDefaultDisableGravatar()
-		disableGravatarSetting = &Setting{SettingValue: strconv.FormatBool(disableGravatar)}
-	} else if err != nil {
-		return err
-	} else {
-		disableGravatar = disableGravatarSetting.GetValueBool()
-	}
-
-	var enableFederatedAvatar bool
-	enableFederatedAvatarSetting, err := GetSetting(ctx, KeyPictureEnableFederatedAvatar)
-	if IsErrSettingIsNotExist(err) {
-		enableFederatedAvatar = setting_module.GetDefaultEnableFederatedAvatar(disableGravatar)
-		enableFederatedAvatarSetting = &Setting{SettingValue: strconv.FormatBool(enableFederatedAvatar)}
-	} else if err != nil {
-		return err
-	} else {
-		enableFederatedAvatar = disableGravatarSetting.GetValueBool()
-	}
-
+	disableGravatar := setting_module.Config().Picture.DisableGravatar.Value()
```

Comment:
But these values hasn't read from database?

Reply:
They will. This is a PoC , demo